src/tab/Tab.tsx

```diff
@@ -0,0 +1,48 @@
+import "./_tab.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import Button from "../button/Button";
+
+export type TabItem = {
+  id: string;
+  children: React.ReactNode;
+  icon?: React.ReactNode;
+};
+
+interface TabProps {
+  items: TabItem[];
+  children: React.ReactNode[];
+  customClassName?: string;
+}
+
+function Tab({items, children, customClassName}: TabProps) {
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
```

Comment:
I think it might be useful if we could pass down a prop to control the initial value of the  `activeTabIndex` state. It can default to `0`